pages/pipelines/build_matrix.md.erb

```diff
@@ -41,6 +41,11 @@ steps:
 {: codeblock-file="pipeline.yml"}
 
 
+<div class="Docs__troubleshooting-note">
```

Comment:
@mcncl The bad news is your PR got caught in the gap between me merging an update to how we do notes, which we now call Callouts: https://github.com/buildkite/docs/blob/main/styleguide/STYLE.md#callouts

The good news is we now get to test the instructions in the `STYLE.md` by updating this to match.

Would you mind trying that for me, please?